tests/common.py

```diff
@@ -0,0 +1,51 @@
+"""Common functions for tests. For pytest fixtures, see conftest.py."""
+
+import os
+import platform
+import shutil
+from pathlib import Path
+
+
+def get_libmf6() -> Path:
+    """Get libmf6 based on environment variable or PATH with mf6.
+
+    To specify directly, set LIBMF6 as a path to the library to test.
+
+    Otherwise, find the mf6 executable from the PATH environment, and
+    try to find the library in the same directory as the executable.
+
+    Additionally, MODFLOW_BIN_PATH as used by modflowpy/install-modflow-action
+    takes priority in searching for the mf6 executable.
```

Comment:
```suggestion
    Otherwise, check for the environment variable `MODFLOW_BIN_PATH` as used by
    modflowpy/install-modflow-action to find the mf6 executable.
    
    Lastly, find the mf6 executable from the PATH environment, and
    try to find the library in the same directory as the executable.
```